cves/CVE-2016-8747.yml

```diff
@@ -43,7 +43,7 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: Http11InpouutBuffer.java allowed remote attackers to read data (information discloure) that was intended for a different request. 
```

Comment:
How did this java file allow attackers to read data?

Reply:
It wasn't specifically described in any article. However, from piecing articles together it seems like the data was exposed and not properly encrypted in the buffer.